templates/adp-backend-template-node/template.yaml

```diff
@@ -298,82 +298,102 @@ spec:
         owner: ${{ parameters.repo_url | parseRepoUrl | pick('owner') }}
         permission: push
 
+    - id: registerFluxService
+      name: Register Service in Flux config
+      action: http:backstage:request
+      input:
+        method: POST
+        path: /proxy/adp-portal-api/FluxTeamConfig/${{ parameters.owner | parseEntityRef | pick('name') }}/services
+        headers:
+          content-type: "application/json"
+        body:
+          name: ${{ parameters.component_id }}
+          isFrontEnd: false
+          environments:
+            - SND3
+            - DEV1
+            - TST1
+            - TST2
+            - PRE1
+            - PRD1
+          configVariables: {}
+
     - id: configureAdoProject
       name: Configure Azure DevOps Project
       action: http:backstage:request
       input:
         method: PATCH
         path: /proxy/adp-portal-api/AdoProject/${{ parameters.ado_project }}/onboard
         headers:
-          content-type: 'application/json'
+          content-type: "application/json"
         body:
           environments:
-          - name: SND3
-            description: SND environment
-          - name: DEV1
-            description: DEV1 environment
-          - name: TST1
-            description: TST1 environment
-          - name: TST2
-            description: TST2 environment
-          - name: PRE1
-            description: PRE1 environment
-          - name: PRD1
-            description: PRD1 environment
+            - name: SND3
+              description: SND environment
+            - name: DEV1
+              description: DEV1 environment
+            - name: TST1
+              description: TST1 environment
+            - name: TST2
+              description: TST2 environment
+            - name: PRE1
+              description: PRE1 environment
+            - name: PRD1
+              description: PRD1 environment
           serviceConnections:
-          - AZD-ADP-SND1
-          - AZD-ADP-SND2
-          - AZD-ADP-SND3
-          - AZD-ADP-SSV3
-          - AZR-ADP-DEV1
-          - AZR-ADP-TST1
-          - AZR-ADP-PRE1
-          - AZD-ADP-SNYK
-          - Defra-ADP-Github
-          - DEFRA-ADP-SonarCloud
+            - AZD-ADP-SND1
+            - AZD-ADP-SND2
+            - AZD-ADP-SND3
+            - AZD-ADP-SSV3
+            - AZR-ADP-DEV1
+            - AZR-ADP-TST1
+            - AZR-ADP-PRE1
```

Comment:
Can we add Prod env connection as well?

Reply:
Yes we can, if it exists now @rajeshkaremane  and wont cause a deploy/setup error

Reply:
can we confirm that

Reply:
I think this is out of scope for this PR, it should be done in another one and against its own ticket. This only changed because prettier fixed the indentation issues on these lines, I havent actually changed anything to do with the environments

Reply:
agreed

Reply:
@LoreSquill  to create separate ticket